assets/sass/print.scss

```diff
@@ -0,0 +1,145 @@
+@media print {
+  .inner {
+    // The `position` of the `inner` class is defined as `relative`, which
+    // causes funny issues when printing, for example tens of empty pages in
+    // the middle. Let's suppress that.
+    position: inherit;
+
+    width: inherit;
+    margin-bottom: 0;
+  }
+
+  #main {
+    margin-bottom: 0;
+
+    /* Improve readability */
+    line-height: 1.4;
+    text-align: justify;
+
+    /* Ensure it uses full page width */
+    width: 100%;
+    border: none;
+    padding: 0;
+  }
+
+  footer {
+    padding: 0;
+    margin-top: 0;
+  }
+
+  aside, .sidebar-btn, #search-container, #reference-version, #dark-mode-button, .site-source {
+    display: none;
+  }
+
+  section {
+    break-inside: avoid-page;
+  }
+
+  div#main {
+    box-decoration-break: clone;
+  }
+
+  .pdf-link, #dark-mode-button {
+    display: none !important;
+  }
+
+  .cheat-sheet {
+    font-size: 8pt;
```

Comment:
how do you think about putting this code in the print stylesheet vs the `cheat-sheet.scss` style sheet?

I had a similar question about dark mode stuff -- for me if I'm editing the "cheat sheet" page it feels easier if all of the CSS for that page (dark mode, print, etc) are in the same place. But I think there are some advantages to having a "dark mode" or "print" stylesheet that have all of the dark mode  print settings that I'm not thinking of.